src/azure-cli/azure/cli/command_modules/role/tests/latest/test_role.py

```diff
@@ -455,7 +455,7 @@ def test_role_assignment_handle_conflicted_assignments(self, resource_group):
         with mock.patch('azure.cli.command_modules.role.custom._gen_guid', side_effect=self.create_guid):
             user = self.create_random_name('testuser', 15)
             self.kwargs.update({
-                'upn': user + '@azuresdkteam.onmicrosoft.com',
+                'upn': user + '@detienne001gmail.onmicrosoft.com',
                 'nsg': 'nsg1'
```

Comment:
Same as above

Reply:
I do not have the permissions to execute this test, please do.